tools/windows/install-help/cal/TargetMachine.cpp

```diff
@@ -82,19 +82,10 @@ DWORD TargetMachine::DetectMachineType()
     DWORD status = NetServerGetInfo(nullptr, 101, reinterpret_cast<LPBYTE *>(&serverInfo));
     if (status != NERR_Success)
     {
-        /*
-         * If the function fails, the return value can be one of the following error codes.
-         *   - ERROR_ACCESS_DENIED
-         *  The user does not have access to the requested information.
-         *  -  ERROR_INVALID_LEVEL
-         *  The value specified for the level parameter is invalid.
-         *  - ERROR_INVALID_PARAMETER
-         *  The specified parameter is invalid.
-         *  - ERROR_NOT_ENOUGH_MEMORY
-         *  Insufficient memory is available.
-         */
-        WcaLog(LOGMSG_STANDARD, "Failed to get server info: %d %d", status, GetLastError());
-        return status;
+        // NetServerGetInfo failed. This is most likely due to the Server service not running, which we can safely ignore.
+        WcaLog(LOGMSG_STANDARD, "Failed to get server info: %d %d. Continuing assuming type is SV_TYPE_WORKSTATION.", status, GetLastError());
+        _serverType = SV_TYPE_WORKSTATION;
+        return ERROR_SUCCESS;
```

Comment:
It was `NERR_ServerNotStarted`in the case we saw, but given that error is undocumented, who knows which other udocumented errors this could be returning... probably `NERR_ServiceNotInstalled` and `NERR_WkstaNotStarted` and maybe more. They all seem non-fatal to me, so I though it would be better to blanket ignore them all.

Reply:
I'll do that on a separate PR, to keep this change small (in case we want to squeeze it in the release).